src/main/java/com/easyrest/components/menuPage/MenuItem.java

```diff
@@ -0,0 +1,26 @@
+package com.easyrest.components.menuPage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class MenuItem {
+    private WebDriver driver;
+
+    private final By addProduct = By.xpath("//button[@aria-label = 'Add to cart']");
+
+    public MenuItem(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public MenuItem addProduct() {
+        List<WebElement> elements = driver.findElements(addProduct);
+        if (elements.size() > 0) {
+            WebElement addition = elements.get(0);
+            addition.click();
```

Comment:
Addition -> A good rule of thumb is: If a name requires a comment, then it doesn't reveal the intent.

Reply:
addition - element that is added to the order

Reply:
What name can you suggest? Because I think that's the name is meaningful

Reply:
for example product